hw2/css/style.css

```diff
@@ -0,0 +1,122 @@
+body {
+	background-color: #f9f9f9;
+	font-family: OpenSans, Ubuntu, Antic, sans-serif;
+} 
+.wrapper {
+	margin: auto;
+	padding: 50px 48px 60px 46px;
+	width: 1060px;
+	background-color: #ffffff;
+	box-shadow: 0 5px 5px rgba(0, 0, 0, 0.3);
+}
+/*HEADER*/
+.top-menu {
+	display: flex;
+	background-color: #ffffff;
+	background-image: linear-gradient(to top, #444343 0%, #5e5e5e 100%); 
+	margin-top: 40px;
+}
+.top-menu a {
```

Comment:
use classes, don't use fat selectors, when there is other way. Use top-menu__link